missions/W4,5_CommonModules /dockerContainerDelete.sh

```
@@ -0,0 +1,10 @@
+#!/bin/sh
+SLAVE_CNT=4
+
+for ((num=0; num<SLAVE_CNT; num++))
```

Shellcheck (0.10.0)

[warning] In POSIX sh, arithmetic for loops are undefined.

(SC3005)